modules/client/src/client-common/utils.ts

```diff
@@ -121,3 +126,50 @@ export function votingModeFromContracts(votingMode: number): VotingMode {
       throw new InvalidVotingModeError();
   }
 }
+
+export function isFailingProposal(actions: DaoAction[]): boolean {
+  const functionNames: string[] = [];
+  // store the function names of the actions
+  for (const action of actions) {
+    try {
+      const fragment = getFunctionFragment(
+        action.data,
+        FAILING_PROPOSAL_AVAILABLE_FUNCTION_SIGNATURES,
+      );
+      console.log(fragment);
+      functionNames.push(
+        fragment.name,
+      );
+    } catch (e) {
+      console.log(e);
```

Comment:
Why?

Reply:
Because if the function signature is not in the available ones it'll throw

Reply:
If a given action is unknown to us, that's not an error. We should only be printing exceptionsl situations if they are crucial to prevent an error or to help diagnosing what went wrong. 
If the `catch` should simply continue iterating, then I would leave `catch() {}`